### PR TITLE
wulan_landing_page

### PR DESCRIPTION
**Pull request by lanabit**

### CODE REVIEW

File: apps/web/src/app/loading.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+          <animateTransform
+            attributeName="transform"
+            type="rotate"
+            dur="2s"
+            values="0 12 12;360 12 12"
+            repeatCount="indefinite"
+          />
+        </g>
+      </svg>
+      {/* <svg
```
Comment (masdefry) [inline apps/web/src/app/loading.tsx:48]: Comment ini masih kepakek? Kalo enggak, dihapus aja ya

**Review by masdefry — COMMENTED** [review]
Masih ada conflict. Coba di resolve dulu ya

File: apps/web/src/app/loading.tsx (hunk 1 above)
Comment (lanabit) [reply apps/web/src/app/loading.tsx:48]: kak, katanya conflicts are too complex to resolve in github. aku coba buka di github desktop dapet alert "branchname already exists". berarti resolvenya harus pake command line kak?

Comment (lanabit) [reply apps/web/src/app/loading.tsx:48]: Kak katanya conflicts too complex to resolve on github, aku resolve pake command line ya?

**Review by ditonatael — APPROVED** [review]

**Review by masdefry — APPROVED** [review]

**Pull request merged**